emscripten-version.txt

```diff
@@ -1 +1 @@
-"1.38.40"
+"1.38.41"
```

Comment:
Should we be changing this?

Reply:
This needs to be changed as part of the PR, since new macros are added to struct_info.json, which triggers rebuild only on version change.

Reply:
@juj we've actually changed that after moving to the new CI.

It's true that locally a version change will clear the cache. But we now do versioning outside of the emscripten repo, see https://github.com/emscripten-core/emscripten/blob/incoming/docs/process.md#minor-version-updates-1xy-to-1xy1 - basically, the emsdk repo has a hash into the releases repo, and that says which hash in emscripten is which version. We do update the versions in emscripten after those updates, but only afterwards.

So @quantum5 is right, we shouldn't change that in this PR. Not changing it won't cause breakage on CI or on emsdk builds, the only risk is when running locally and separately from the emsdk.

Reply:
oh gotcha, I'll revert